### PR TITLE
Species Overhaul

### PR DESCRIPTION
**Pull request by LetterJay**
This is to fastforward all the species work to the more recent build of /tg/. I'd say it's already majority of the way there. Tested it, works.

Still have a few more things to do.

- [x] Humans
- [x] Ashlizards
- [x] Lizards
- [x] Podpeople
- [x] Canids
- [x] Felid
- [ ] ~~Xenos~~

### CODE REVIEW

**Comment by LetterJay** [comment]
Vore seems to be working. Feel free to review and test. Species work from what I see.

File: code/modules/mob/living/carbon/human/human_defense.dm [hunk 1]
```diff
@@ -160,8 +160,8 @@
 	return ..()
 
 /mob/living/carbon/human/grabbedby(mob/living/carbon/user, supress_message = 0)
-	if(user == src && pulling && !pulling.anchored && grab_state >= GRAB_AGGRESSIVE && (disabilities & FAT) && ismonkey(pulling))
-		devour_mob(pulling)
+	if(user == src && pulling && !pulling.anchored && grab_state >= GRAB_AGGRESSIVE && iscarbon(pulling))
+		vore_attack(user, pulling)
```
Comment (Iamgoofball) [inline code/modules/mob/living/carbon/human/human_defense.dm:164]: oh jesus christ what the fuck

Comment (ghost) [reply code/modules/mob/living/carbon/human/human_defense.dm:164]: what the fuck are you doing here

File: code/modules/mob/living/carbon/human/species_types.dm [hunk 2]
```diff
@@ -1481,3 +1484,164 @@ SYNDICATE BLACK OPS
...
+/datum/species/yautja/before_equip_job(datum/job/J, mob/living/carbon/human/H, visualsOnly = FALSE)
+	var/datum/outfit/yautja_basic/O = new /datum/outfit/yautja_basic//Just basic gear. Doesn't include anything that gives any meaningful advantage.
+	H.equipOutfit(O, visualsOnly)
+	return 0*/
+
+/datum/species/octopus
+	blacklisted = 1
+/datum/species/carp
+	blacklisted = 1
+/datum/species/horse
```
Comment (Iamgoofball) [inline code/modules/mob/living/carbon/human/species_types.dm:1630]: uh, no

Comment (ghost) [reply code/modules/mob/living/carbon/human/species_types.dm:1630]: a

File: code/modules/mob/living/carbon/human/human_defense.dm (hunk 1 above)
Comment (Iamgoofball) [reply code/modules/mob/living/carbon/human/human_defense.dm:164]: i got here from a vg issue

vg has some explaining to do

**Comment by ghost** [comment]
can you not shitpost here

**Pull request merged**